src/redux/thunk.js

```diff
@@ -6,6 +6,7 @@ import {
   keepSession,
   getUsers,
   createUsers,
+  deleteUser,
```

Comment:
please add 'updateUser' here as well. I think you accidentally took it out during reformatting